src/ui/component/fileListItem/view.jsx

```diff
@@ -110,7 +110,7 @@ function FileListItem(props: Props) {
           {!slim && (
             <div>
               {isChannel && <SubscribeButton uri={uri.startsWith('lbry://') ? uri : `lbry://${uri}`} />}
-              <FileProperties uri={uri} />
+              {!isChannel && <FileProperties uri={uri} />}
```

Comment:
channels aren't files

if this component is now rendering streams and channels, I'd rename it `ClaimListItem` or similar

Reply:
Nice good call. Will rename all of these to `claimX`